prody/proteins/pdbfile.py

```diff
@@ -590,16 +593,34 @@ def _parsePDBLines(atomgroup, lines, split, model, chain, subset,
             resnames[acount] = resname
             chainids[acount] = chid
             if isPDB:
-                try:
-                    resnums[acount] = int(line[22:26])
-                except ValueError:
+                if dec:
+                    try:
+                        resnum = int(line[22:26])
+                    except ValueError:
+                        dec = False
+
+                    if resnums[acount-2] >= MAX_N_RES:
```

Comment:
Same as below. Make sure you check bounds whenever you do subtractions on indices. Why `acount-2` here?

Reply:
Because it's been incremented already and `acount-1` is the current atom. It wasn't good though because multiple atoms could have the same resnum. This has been fixed in the recent commit.